lib/config.atd

```diff
@@ -20,6 +20,7 @@ type prefix_rules = {
 (* This type of rule is used for PR and issue notifications. *)
 type label_rules = {
   ?default_channel: string nullable; (* if none of the rules is matching *)
+  ?ignored_users: string list nullable; (* list of ignored users *)
```

Comment:
i don't think it should be in label_rules, it is global property

Reply:
also can make default value empty list which is good semantics and simplifies code slightly

Reply:
> also can make default value empty list which is good semantics and simplifies code slightly

ah sounds good will change!

Reply:
> i don't think it should be in label_rules, it is global property

my rationale was since we only needed to filter out PR comments, we should put the list in labels as those are the rules related to PRs (and we would want to be notified if some user like a bot managed to push a commit).

Reply:
I also think that it should be a more global property. We might want to ignore some commit comments for example, and those can happen outside of a PR. But I'm not sure what's the right place to put it

Reply:
wait i'm confused so we want to ignore commit comments from that user as well?

Reply:
I'm thinking probably any kind of comment from that user yes

Reply:
and maybe we can have a dedicated `ignore` object at the root of the config. For now we are ignoring comments from a user, but later we could decide to commits for that user too or ignore some branches or some labels or paths or ...

could be something like this (just throwing an idea, no really sure)

```json
{
  ignore: {
    comments: ["user_a", "user_b"]
  }
}
```

Reply:
okay sounds good--i'll do ignore everything for now then!

Reply:
okay user is ignored globally now!